slp_abacus/tests/util/builders.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from slp_drawio.slp_drawio.objects.diagram_objects import DiagramComponent, Diagram, DiagramTrustZone, \
```

Comment:
think it is another copy-paste from drawio. This is an auxiliar class to build objects for testing purposes. Once we create the Abacus tests, it is important to fix this import to use the `slp_abacus` objects instead of the `slp_drawio` ones. Otherwise, you will get errors.